sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -194,12 +194,6 @@ internal static ExtractKeyPhrasesResultCollection ConvertToExtractKeyPhrasesResu
         internal static List<CategorizedEntity> ConvertToCategorizedEntityList(List<Entity> entities)
             => entities.Select((entity) => new CategorizedEntity(entity)).ToList();
 
-        //internal static CategorizedEntityCollection ConvertToCategorizedEntityCollection(DocumentEntities documentEntities)
-        //{
-        //    return new CategorizedEntityCollection(ConvertToCategorizedEntityList(documentEntities.Entities.ToList()), ConvertToWarnings(documentEntities.Warnings));
-        //}
-
-        // CHANGED: added overload for New Swagger
         internal static CategorizedEntityCollection ConvertToCategorizedEntityCollection(EntitiesResultDocumentsItem documentEntities)
         {
             List<TextAnalyticsWarning> warnings = new List<TextAnalyticsWarning>();
```

Comment:
you already have a method to do this https://github.com/Azure/azure-sdk-for-net/blob/871dc09e0da54714390e0e4f6b0ce2d76ca1940c/sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs#L83

No need to do it again.

Reply:
dang....  I missed this.   I'm a dirty hack who will go sit in the corner now.

Reply:
No need :( fresh eyes always help